src/Apps/Artwork/Components/PricingContext.tsx

```diff
@@ -38,78 +41,115 @@ function PricingContext({ artwork }: PricingContextProps) {
-        }
-        bars={artwork.pricingContext.bins.map(
-          (bin): BarDescriptor => {
-            const binMinPrice = bin.minPrice != null ? bin.minPrice : "$0"
-            const title = `${binMinPrice}–${bin.maxPrice}`
-            const artworkFallsInThisBin =
-              priceCents >= bin.minPriceCents && priceCents < bin.maxPriceCents
+  render() {
+    const { artwork } = this.props
 
-            const binValue =
-              artworkFallsInThisBin && bin.numArtworks === 0
-                ? 1
-                : bin.numArtworks
-            const labelSuffix = binValue === 1 ? " work" : " works"
+    if (!artwork.pricingContext) {
+      return null
+    }
 
-            return {
-              value: binValue,
-              label: {
-                title,
-                description: binValue + labelSuffix,
-              },
-              onClick: openCollectPage(
-                bin.minPriceCents,
-                bin.maxPriceCents,
-                artwork.category,
-                artwork.widthCm,
-                artwork.heightCm,
-                artwork.artists[0].id
-              ),
-              highlightLabel: artworkFallsInThisBin
-                ? {
-                    title,
-                    description: "This work",
-                  }
-                : undefined,
-            }
+    const priceCents = artwork.priceCents.max
+      ? (artwork.priceCents.min + artwork.priceCents.max) / 2
+      : artwork.priceCents.min
+    return (
+      <BorderBox mb={2} flexDirection="column">
+        <Waypoint onEnter={once(this.trackImpression.bind(this))} />
+        <Sans size="2" weight="medium">
+          Price
+        </Sans>
+        <Flex>
+          <Sans size="2">
+            Price ranges of {artwork.pricingContext.filterDescription}
+          </Sans>
+          <PricingContextModal />
+        </Flex>
+        <Spacer mb={[2, 3]} />
+        <BarChart
+          minLabel={
+            artwork.pricingContext.bins[0].minPrice != null
+              ? artwork.pricingContext.bins[0].minPrice
+              : "$0"
+          }
+          maxLabel={
+            artwork.pricingContext.bins[artwork.pricingContext.bins.length - 1]
+              .maxPrice + "+"
           }
-        )}
-      />
-    </BorderBox>
-  )
+          bars={artwork.pricingContext.bins.map(
+            (bin): BarDescriptor => {
+              const binMinPrice = bin.minPrice != null ? bin.minPrice : "$0"
+              const title = `${binMinPrice}–${bin.maxPrice}`
+              const artworkFallsInThisBin =
+                priceCents >= bin.minPriceCents &&
+                priceCents < bin.maxPriceCents
+
+              const binValue =
+                artworkFallsInThisBin && bin.numArtworks === 0
+                  ? 1
+                  : bin.numArtworks
+              const labelSuffix = binValue === 1 ? " work" : " works"
+              return {
+                value: bin.numArtworks,
```

Comment:
Is this a regression?

Reply:
Idk I had really bad merge conflict but I thought I fixed it